From 0651c4edc9909832ff2acbbf170268f34e36616d Mon Sep 17 00:00:00 2001
From: Fabio Estevam <festevam@gmail.com>
Date: Mon, 8 Jan 2024 15:00:01 -0300
Subject: [PATCH] cube-gears: Change header file to <GLES3/gl3.h>

Since commit 96d63eb59e34 ("kmscube: Add gears mode"), kmscube fails
to build on platforms without <GL/gl.h>.

Fix it by changing the header file to <GLES3/gl3.h>.

Reported-by: Martin Jansa <martin.jansa@gmail.com>
Suggested-by: Martin Jansa <martin.jansa@gmail.com>
Signed-off-by: Fabio Estevam <festevam@gmail.com>
Upstream-Status: Submitted [https://gitlab.freedesktop.org/mesa/kmscube/-/merge_requests/51]
---
 cube-gears.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cube-gears.c b/cube-gears.c
index d5b7a5f9cbef..cb538ecc4aee 100644
--- a/cube-gears.c
+++ b/cube-gears.c
@@ -31,7 +31,7 @@
 #include <sys/time.h>
 #include <math.h>

-#include <GL/gl.h>
+#include <GLES3/gl3.h>

 #include "common.h"
 #include "esUtil.h"
--
2.34.1
